### ISSUE TITLE
student.predict instead of teacher.predict in the func train_distillation

### ISSUE THREAD

**Issue by IvanKush** [carlini/nn_robust_attacks#2]
should be `student.predict(data.train_data)` instead of `teacher.predict(data.train_data)` after `student` training in the func `train_distillation`?

corrected code:
```
 # train the student model at temperature t
    student = train(data, file_name, params, num_epochs, batch_size, train_temp,
                    init=file_name+"_init")

    # and finally we predict at temperature 1
    predicted = student.predict(data.train_data)

    print(predicted)
```

**Comment by carlini** [comment]
Yep, good catch. This shouldn't affect any results, just change the print output.

**Comment by carlini** [comment]
Fixed by 1e8dd2d85df7f616230688eeb7d71cb363baa97d.
